evalml/pipelines/components/transformers/preprocessing/datetime_featurizer.py

```diff
@@ -7,19 +7,33 @@
 
 
 def _extract_year(col):
-    return col.dt.year
+    return col.dt.year, None
```

Comment:
I debated whether or not we should encode the years starting at 0 but that sounds like a performance test? I can file an issue but it'd be blocked by #1383

Reply:
Would encoding years take away contextual information about the datetime later on in the pipeline? January will always be mapped to 0 but 1981 being mapped to 0 would take information wouldn't it? Or maybe I'm misunderstanding!

Reply:
I had a thought that linear models would work best with scaled features. Now that I think about it again, we automatically add a standard scaler for linear models at least in AutoML so maybe we can keep as is for now!